hugo/content/contact.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contact
+toc: true
+---
+
+# Contact
+
+You can contact us in several ways:
+
+## Community Channels
+
+* Our [issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
+* [Discussions on GitHub](https://github.com/mumble-voip/mumble/discussions)
+* IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
+* Chat: [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
```

Comment:
I think this should also be made more consistent. Chat is very unspecific, and IRC is also a chat.

With the other bullet points being mostly full links it seems good to make them here so as well.

```suggestion
* [IRC #mumble on freenode](irc://irc.freenode.org/mumble)
* [Mumble chat room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
```

Reply:
You are correct with this, but I would change it differently:
```

* Chat:
    * IRC: bla
    * Matrix: bla
```